### DISCUSSION TITLE
[11일차 - 예진] 오버로딩, 오버 라이딩

### DISCUSSION THREAD

**Discussion by ez1n** [zzangCS/zzangInterview#37] [프론트 면접 질문]
오버 로딩과 오버라이딩에 대해 설명해 주세요.

**Comment by skyhanull** [comment]
메서드 오버 로딩(method overloading)은 메서드의 이름이 같지만 매개변수의 타입을다르게 정의하는 방법을 일컫습니다. 
오버 라이딩(overriding)은 부모 클래스에 정의된 메서드를 자식 클래스에서 새로 구현하는 것을 일컫는 개념입니다.

**Comment by ez1n** [comment]
- **오버로딩**
함수의 이름은 동일하지만 매개변수의 타입 또는 개수가 다른 메소드를 구현하고 매개변수에 의해 메소드를 구별하여 호출하는 방식
자바스클비트는 지원하지 않음!!
    
- **오버라이딩**
상위 클래스가 가지고 있는 메소드를 하위 클래스가 재정의하여 사용하는 방식
